.github/workflows/release.yaml

```diff
@@ -14,20 +14,19 @@ jobs:
 
     steps:
       - name: Checkout repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4.2.2
         with:
           fetch-depth: 0 # fetch all commits/branches so that mike works
 
       - name: Set up Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5.6.0
         with:
-          python-version: 3.9
+          python-version: 3.10
 
       - name: Install Poetry
-        uses: snok/install-poetry@v1.3
-        with:
-          virtualenvs-create: true
-          virtualenvs-in-project: true
+        run: |
+          pipx install poetry
+          pipx inject poetry poetry-dynamic-versioning
```

Comment:
Again. The repo is not yet setup for dynamic-versioning so you don't need to install it.

Reply:
I'm not sure if this was a particular design decision to be honest.  It could be that linkml-model repo was created before we started using dynamic versioning and we didn't bring it back up to "standard." 

We can do dynamic versioning in another PR.

Reply:
I would also suggest to add it in another PR.